Makefile

```diff
@@ -1,9 +1,17 @@
 NAME    := unsee
 VERSION := $(shell git describe --tags --always --dirty='-dev')
+GO      := GO15VENDOREXPERIMENT=1 go
+PROMU   := $(GOPATH)/bin/promu
+pkgs     = $(shell $(GO) list ./... | grep -v -E '/vendor/')
+
+PREFIX                  ?= $(shell pwd)
+BIN_DIR                 ?= $(shell pwd)
+DOCKER_IMAGE_NAME       ?= unsee
```

Comment:
There'a already a target for building docker images, there's no need for an extra variables for it